sox.features.js

```diff
@@ -705,28 +705,34 @@
         $(document.getElementById('question-header')).prepend(getHotDiv());
       }
 
-      if (sox.location.on('/questions') || $('.question-summary').length) {
-        const proxyUrl = 'https://cors-anywhere.herokuapp.com/'; //CORS proxy
-        const hnqJSONUrl = 'https://stackexchange.com/hot-questions-for-mobile';
-        const requestUrl = proxyUrl + hnqJSONUrl;
-
-        $.get(requestUrl, results => {
-          if (sox.location.on('/questions/')) {
-            $.each(results, (i, o) => {
-              if (document.URL.indexOf(o.site + '/questions/' + o.question_id) > -1) addHotText();
-            });
-          } else {
-            $('.question-summary').each(function() {
-              const id = $(this).attr('id').split('-')[2];
-              if (results.filter(d => {
-                return d.question_id == id;
-              }).length) {
-                $(this).find('.summary h3').prepend(getHotDiv('question-list'));
-              }
-            });
-          }
-        });
+      const sitename = sox.site.currentApiParameter;
+
+      if (sox.location.on('/questions')) {
+        const postId = window.location.pathname.split('/')[2]
+        apiCall(postId, sitename);
+      } else if ($('.question-summary').length) {
+          $('.question-summary').each(function() {
+              const postID = $(this).attr('id').split('-')[2];
+              apiCall(postID, sitename, this);
+          });
       }
+        function apiCall(postID, sitename, el) {
```

Comment:
Could you reindent this function to make it line up with the previous `if`, and have the actual function use 2-space indents rather than 4? (just to match with the rest of the code!)

Reply:
Oops! Tampermonkey's fault! It just used 4-space indents for everything inside a function/loops/statements, etc. Fixed now, check again.